src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -329,10 +327,8 @@ private static bool ValidateInternalRecursive(LicenseContext context, Type type,
                     if (license != null)
                     {
                         license.Dispose();
-                        license = null;
```

Comment:
If ValidateInternalRecursive throws before assigning to `license`, then this null assignment is semantic.  It should probably come back (and if the analyzer is reporting it, we should fix the analyzer).

Reply:
Yep, I fixed this one in previous commit just didn't push on time